gamedb.yaml

```diff
@@ -47,6 +47,12 @@
   notes:
     - Displays keyboard icons
 
+- name: Brothers - A Tale of Two Sons
+  platform: epic-store
+  id: Tamarind
+  banner: https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/f730dba64b56c5b89a94bcb8cc3b0fba.jpg
```

Comment:
Would be better to use the steam banner. An upcoming feature will automatically grab all the other types of artwork from Steam in addition to the horizontal banner.